src/current/cockroachcloud/authorization.md

```diff
@@ -146,100 +117,56 @@ Users with this role can view cluster details, allowing them to [export a connec
-
-When all role assignments have been removed for a user, they still implicitly have the Org Member role which is granted to each newly-added {{ site.data.products.db }} member, and the member is not automatically removed from the organization. Refer to: [Remove a team member](managing-access.html#remove-a-team-member)
-
-### Which roles grant the ability to add, remove, and manage members in in a {{ site.data.products.db }} organization?
-
-Users with the Org Administrator (legacy) role are allowed to manage users and roles at both the organization and the cluster scopes. Users with the Cluster Admin role are only allowed to manage role assignments at the cluster scope.
-
-### What is the Cluster Creator role useful for when there’s a Cluster Admin role as well?
-
-A user with the Cluster Creator role can create new clusters in the {{ site.data.products.db }} organization, so this role can be assigned only at the organization scope.
-
-After the cluster is created, its creator is automatically granted the Cluster Admin role on that cluster. If that user already had the Cluster Admin role at the organization scope, this cluster-specific grant appears to have no effect.
-
-This overlap allows admins to give users from different projects or teams access to create and fully manage their own clusters without the ability to manage clusters owned by other projects or teams. For example, two different users from different teams could each be granted the Cluster Creator role so that they can fully manage clusters they own but not clusters owned by anyone else.
+## Service accounts
 
-### Are SQL roles part of the {{ site.data.products.db }} authorization model?
+Service accounts authenticate with API keys to the {{ site.data.products.db }} API, rather than to the {{ site.data.products.db }} Console UI.
 
-{{ site.data.products.db }} has a two-level authorization model:
+Service accounts operate under a unified authorization model with organization users, and can be assigned all of the same [organization roles](#organization-user-roles) as users, but note that some actions are available in the console but not the API, or vice versa (For example, in the [Cluster Operator Role](#cluster-operator)).
 
-1. SQL level in a cluster: Each CockroachDB cluster has its own set of SQL users and roles defined in it. Roles grant users permission to execute some set of SQL statements against some set of database resources (like tables, databases) on the cluster.
-2. Organization level: Each {{ site.data.products.db }} organization has a set of roles defined in it, which allow users to perform administrative tasks relating to the management of clusters, organization users, SQL users, and billing.
+*Legacy service accounts* that were created before the updated authorization model was enabled for your cloud organization may have roles assigned under the *legacy model*:
```

Comment:
Do you think it is worthwhile to separate off the Legacy service account description and bullets a bit with an H3 just before the italics "Legacy service accounts"? Maybe not, just a thought. If that was something you did, could also move the "Refer to Manage Service Accounts" above that H3 heading

Reply:
i like it